internal/modifier/templates/panic_error.go.tmpl

```diff
@@ -0,0 +1,55 @@
+{{- define "panicError" -}}
+{{- $cff := import "go.uber.org/cff" -}}
+{{ template "stacktrace" }}
+err = {{ $cff }}.PanicError{
+    Value:      recovered,
+    Stacktrace: stacktrace,
+}
+{{- end -}}
+
+{{- define "stacktrace" -}}
+{{- $runtime := import "runtime" -}}
+{{ template "funcFormatFunction" }}
+{{ template "funcGetStack" }}
+pc := make([]uintptr, 64)
+{{- /* skipping 2 in the callers, which are: */ -}}
+{{- /* - runtime.Callers */ -}}
+{{- /* - caller of Callers (self) */ -}}
+{{- /* because the panic should not be in any of these callers */}}
+n := {{ $runtime }}.Callers(2, pc)
+stacktrace := "[frames]:\n"
+if n != 0 {
+    pc = pc[:n]
+    frames := {{ $runtime }}.CallersFrames(pc)
+    stacktrace = getStack(frames)
+}
+{{- end -}}
+
+{{- define "funcGetStack" -}}
+{{- $fmt := import "fmt" -}}
+{{- $strings := import "strings" -}}
+getStack := func (frames *{{ import "runtime" }}.Frames) string {
+    var b {{ $strings }}.Builder
+	{{ $fmt }}.Fprintf(&b, "[frames]:\n")
+    for {
+        frame, more := frames.Next()
+        {{ $fmt }}.Fprintf(&b, "%s()\n\t%s:%d\n", formatFunction(frame.Function), frame.File, frame.Line)
+        {{- /* Check whether there are more frames to process after this one */ -}}
+        if !more {
+            break
+        }
+    }
+    return b.String()
+}
```

Comment:
Are you sure we need to do all of this ourselves?

https://pkg.go.dev/runtime/debug#Stack generates a formatted stack trace with a large enough buffer to capture everything. Yeah, it'll also include this code, but that's not the end of the world.

Minus that, even if we need all this logic, I'm not sure all this needs to be in a template? Doesn't look like the template makes use of anything flow specific. This could all be a function exported by the cff package that's imported and used by the template.